layouts/partials/social.html

```diff
@@ -16,10 +16,13 @@
     {{ range $i, $social :=  .Site.Params.social_static }}
       {{ if or ($len_socials) (ne $i 0) }} &nbsp;&ndash;&nbsp; {{ end }}
       <a class="social-icon" href="{{ $social.url }}" name="{{ $social.name }}">
-        <img
-          src="{{ $base }}images/{{ $social.icon }}"
-          alt="{{ $social.name }}"
-        />
+        <object
+          type="image/svg+xml"
```

Comment:
It doesn't work in case web pages use the local social image as `.png`.

I think it would be better to add another option in the configuration file and judge it by that like below.

`config.toml`

```toml
[[params.social_static]]
  name = "example"
  icon = "example.svg"
  type = "svg"
```

And `social.html`

```html
{{ if eq $social.type "svg" }}
  <!-- object tag (this change) -->
{{ else }}
  <!-- img tag -->
{{ end }}
```